src/bookshelf/src/bksw_app.erl

```diff
@@ -26,6 +26,7 @@
 %% ===================================================================
 
 start(_StartType, _StartArgs) ->
+    os:set_signal(sigterm, default),
```

Comment:
It might be useful to add a comment explaining the need for this change.

Reply:
REF: https://github.com/erlang/otp/commit/4c408d98127c3a44f47b86e89d2a0bd5567b1e70#diff-17c71c5b10abe84f2b086c62917f1e33

Reply:
Added comments to source files.